packages/dashboard/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "rootDir": "./",
     "outDir": "./dist/"
   },
-  "include": ["./src/", "./stories/", "./src/locales/*.json"],
+  "include": ["./src/", "./stories/", "./src/**/*.json"],
```

Comment:
why

Reply:
I need to import TokenIconSpecialIconList.json